ui/articles-aem-renderer/src/main/java/org/cru/godtools/articles/aem/view_model/ArticleViewModel.java

```diff
@@ -18,6 +20,16 @@
     private final AttachmentRepository mAttchReposistory;
     private final ManifestAssociationRepository mManifestRepository;
 
+    /**
+     * This allows other models to get an instance of this ViewModel with out adding dependencies
+     * to their .gradle file
+     * @param activity FragmentActivity
+     * @return this
+     */
+    public static ArticleViewModel getInstance(FragmentActivity activity){
+        return ViewModelProviders.of(activity).get(ArticleViewModel.class);
```

Comment:
because you are calling this from a fragment you should use the Fragment itself and not the activity it was attached to.